manifests/4.7/elasticsearch-operator.v4.7.0.clusterserviceversion.yaml

```diff
@@ -112,16 +112,29 @@ metadata:
         ]
 spec:
   version: 4.7.0
-  displayName: Elasticsearch Operator
+  displayName: OpenShift Elasticsearch Operator
   minKubeVersion: 1.18.3
 
   description: |
     The Elasticsearch Operator for OCP provides a means for configuring and managing an Elasticsearch cluster for use in tracing and cluster logging.
     This operator only supports OCP Cluster Logging and Jaeger.  It is tightly coupled to each and is not currently capable of
     being used as a general purpose manager of Elasticsearch clusters running on OCP.
 
+    It is recommended that this operator be installed in the `openshift-operators-redhat` namespace to 
+    properly support the Cluster Logging and Jaeger use cases.
+
     Once installed, the operator provides the following features:
-    * **Create/Destroy**: Deploy an Elasticsearch cluster to the same namespace in which the Elasticsearch custom resource is created.
+    * **Create/Destroy**: Deploy an Elasticsearch cluster to the same namespace in which the elasticsearch CR is created.
+    * **Update**: Changes to the elasticsearch CR will be scheduled and applied to the cluster in a controlled manner (most often as a rolling upgrade).
+    * **Cluster health**: The operator will periodically poll the cluster to evaluate its current health (such as the number of active shards and if any cluster nodes have reached their storage watermark usage).
+    * **Redeploys**: In the case where the provided secrets are updated, the Elasticsearch Operator will schedule and perform a full cluster restart.
+    * **Index management**: The Elasticsearch Operator will create cronjobs to perform index management such as roll over and deletion.
+
+    ### Additionally provided features
+    * Out of the box multitenancy that is integrated with OCP user access control.
+    * Document Level Security
+    * mTLS communication between Elasticsearch, Kibana, Index Management cronjobs, and CLO's Fluentd
+    * Prometheus Alerting rules
```

Comment:
Provides OCP dashboards for ES.

Reply:
is this another point to add or is this a correction to the Prometheus Alerting rules? (if its the latter is there a major difference between the two?)

Reply:
Yes dashboards are a different thing than alerting rules.